src/auth.rs

```diff
@@ -207,25 +225,29 @@ impl Authenticator {
             if let Some(&Binary(_, ref payload)) = doc.get("payload") {
                 let payload_str = match String::from_utf8(payload.to_owned()) {
                     Ok(string) => string,
-                    Err(_) => return Err(ResponseError("Invalid UTF-8 payload returned".to_owned()))
+                    Err(_) => {
+                        return Err(ResponseError("Invalid UTF-8 payload returned".to_owned()))
+                    }
                 };
 
                 // Check that the signature exists
                 let verifier = match scan_fmt!(&payload_str[..], "v={}", String) {
                     Some(string) => string,
-                    None => return Err(MaliciousServerError(MaliciousServerErrorType::NoServerSignature)),
+                    None => return Err(
+                        MaliciousServerError(MaliciousServerErrorType::NoServerSignature)),
```

Comment:
Yeah, the line-length things it does do seem pretty odd. Maybe there's a way to run it with the line-length lint turned off?